.circleci/config.yml

```diff
@@ -95,6 +95,22 @@ jobs:
       - run: docker login -u $DOCKER_USER -p $DOCKER_PASS
       - run: make docker-publish version=$CIRCLE_TAG
 
+  "release_cli_dev":
+    <<: *run_on_docker
+    steps:
+      - checkout
+      - run: make build-cmd-all version="dev build `echo $CIRCLE_SHA1 | cut -c1-7`"
+      - run: go get -u github.com/tcnksm/ghr
+      - run: ghr -u mesg-foundation -r engine -delete -prerelease -n "Developer Release" -b "Warning - this is a developer release, use it only if you know what are doing. Make sure to pull the latest \`mesg/engine:dev\` image." release-dev ./bin
```

Comment:
Please put that as makefile target and let's only call make from CI (if it's possible). Same below with `release_cli_prod`

Reply:
Lookin on CI, we can create `install-go` target in Makefile, but not related for this PR